### PR TITLE
Small update to cql4browsers removing commas

### PR DESCRIPTION
**Pull request by jbradl11**
Small update to the cql-execution enigne. This PR is to keep cql4browsers up to date.
Related PR in the cql-execution engine: https://github.com/cqframework/cql-execution/commit/ce30329519ffedf64cbb9264548ec9dd551b1c40

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] Related Jira Ticket: https://jira.mitre.org/browse/BONNIE-1382
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint: N/A

**Reviewer 1:**

Name: @c-monkey
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name: @losborne 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: vendor/assets/javascripts/cql4browsers.js [hunk 1]
```diff
@@ -1,6 +1,3 @@
-/**
```
Comment (c-monkey) [inline vendor/assets/javascripts/cql4browsers.js]: You need to update this comment

Comment (jbradl11) [reply vendor/assets/javascripts/cql4browsers.js]: Good catch! That's what I get for trying to get it done right before leaving work.

**Comment by c-monkey** [comment]
@jbradl11 The front end coverage decreased from:

```
Statements   : 56.95% ( 6823/11981 )
Branches     : 39.97% ( 1791/4481 )
Functions    : 47.24% ( 1120/2371 )
Lines        : 54.69% ( 6106/11165 )
```
to:
```
Statements   : 55.03% ( 7757/14095 )
Branches     : 36.33% ( 1886/5191 )
Functions    : 45.12% ( 1272/2819 )
Lines        : 52.55% ( 6851/13036 )
```

Thereby decreasing our coverage beneath the threshold.

There are no regressions or coverage decreases in the regression tests.

**Comment by losborne** [comment]
For me the `cql_qdm_patientapi` coverage is the same for both branches:
```
Statements   : 63.3% ( 1359/2147 )
Branches     : 33.66% ( 239/710 )
Functions    : 50.94% ( 245/481 )
Lines        : 60.03% ( 1143/1904 )
```

Bonnie `bonnie-v2.1` as it is vs Gemfile modified to point to this branch also show the same coverage for me:
```
Statements   : 56.95% ( 6823/11981 )
Branches     : 39.97% ( 1791/4481 )
Functions    : 47.24% ( 1120/2371 )
Lines        : 54.69% ( 6106/11165 )
```

@c-monkey if you are able to reproduce your results we should look at what is causing the discrepancy.

**Comment by c-monkey** [comment]
@losborne  As a sanity check, I again very deliberately went and did a `bundle exec rake tmp:clear` and still got the same coverage results. Maybe I'm doing something wrong: I commented out the git gem reference and uncommented the relative path reference. Look at the SHA of your referenced cql_qdm_patientapi in the Gemfile.lock, is it `d3f449a7c6a66799396b2c92fa2057562af2c7a5`?
